packages/fast-components-react-base/src/slider-track-item/slider-track-item.tsx

```diff
@@ -0,0 +1,162 @@
+    {}
+> {
+    public static displayName: string = `${DisplayNamePrefix}SliderTrackItem`;
+
+    public static contextType: React.Context<SliderContextType> = SliderContext;
+
+    protected handledProps: HandledProps<SliderTrackItemHandledProps> = {
+        maxValuePositionBinding: void 0,
+        minValuePositionBinding: void 0,
+        managedClasses: void 0,
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLDivElement> {
+        return (
+            <div
+                {...this.unhandledProps()}
+                className={this.generateClassNames()}
+                style={{
+                    position: "absolute",
+                    ...this.applyPositioningValues(),
+                }}
+            >
+                {this.props.children}
+            </div>
+        );
+    }
+
+    /**
+     * Create class-names
+     */
+    protected generateClassNames(): string {
+        let classNames: string = get(this.props, "managedClasses.sliderTrackItem", "");
+
+        if (
+            (this.context as SliderContextType).sliderOrientation ===
+            SliderOrientation.vertical
+        ) {
+            classNames = `${classNames} ${get(
+                this.props,
+                "managedClasses.sliderTrackItem__orientationVertical",
+                ""
+            )}`;
+        } else {
+            classNames = `${classNames} ${get(
+                this.props,
+                "managedClasses.sliderTrackItem__orientationHorizontal",
+                ""
+            )}`;
+        }
+
+        return super.generateClassNames(classNames);
+    }
+
+    /**
+     * Gets the appropriate absolute positioning
+     */
+    private applyPositioningValues = (): any => {
+        const minValue: number = this.getPctValueFromPositionBinding(
+            this.props.minValuePositionBinding
+        );
+        const maxValue: number = this.getPctValueFromPositionBinding(
+            this.props.maxValuePositionBinding
+        );
+
+        if (
+            (this.context as SliderContextType).sliderOrientation ===
+            SliderOrientation.vertical
+        ) {
+            return {
+                top: `${100 - maxValue}%`,
+                bottom: `${minValue}%`,
+            };
+        } else if ((this.context as SliderContextType).sliderDirection === "rtl") {
+            return {
+                left: maxValue !== undefined ? `${100 - maxValue}%` : undefined,
+                right: minValue !== undefined ? `${minValue}%` : undefined,
+            };
+        } else {
+            return {
+                left: minValue !== undefined ? `${minValue}%` : undefined,
+                right: maxValue !== undefined ? `${100 - maxValue}%` : undefined,
+            };
+        }
+    };
+
+    /**
+     *  Returns a percent value to use for positioning based on the selected anchor type
+     */
+    private getPctValueFromPositionBinding = (
```

Comment:
can you rename this as `getPositionAsPercent` or something? There is never a need to use shorthand since this all gets minified.

Reply:
yep